Functions/Administration/xASL_adm_DefineASLSequence.m

```diff
@@ -99,17 +98,17 @@
 % We default to 3D spiral because this sequence has whole-brain coverage
 % and negligible geometric distortion, so we don't do any special
 % readout-specific image processing for this sequence
-if ~isfield(xQ,'Sequence') || isempty(xQ.Sequence)
-    warning('No xQ.Sequence defined');
+if ~isfield(xQ, 'PulseSequenceType') || isempty(xQ.PulseSequenceType)
```

Comment:
I should have changed this when making the population module sequence-independent.
Also note that this is not the same as checking Sequence (2d_epi vs epi).

We can/should remove all lines from 98 until 109.
If we cannot detect the PulseSequenceType and/or MRAcquisitionType, we simply give an error (we would never have this because we create ASL BIDS in our import)

1. In the ASL module, we always need x.Q.Readout. If it is missing, we can at the location where we need it (e.g., susceptibility mask creation) just not mask susceptibility -> the processing stays the same as what we do now, but without adding a potentially incorrect PulseSequenceType & MRAcquisitionType to the provenance.
2. In the population module, we assume that PulseSequenceType & MRAcquisitionType are in the json sidecar. If they are not (which should be a rare case), then we still not want incorrect PulseSequenceType & MRAcquisitionType but the effective resolution calculation part will give a warning and assume nominal resolution == effective resolution.